.github/workflows/ci.yml

```diff
@@ -206,9 +206,9 @@ jobs:
           set -o xtrace
           mingw32-make.exe binaries
           bindir="$(pwd)"
-          SHIM_COMMIT=$(grep 'Microsoft/hcsshim ' go.mod | awk '{print $2}')
+          SHIM_COMMIT=$(grep -m 1 'Microsoft/hcsshim ' go.mod | sed -e 's#.*[ -]##')
```

Comment:
Why this?

Reply:
Two reasons:
* This command fails if the hcsshim being pointed to is a commit, not a tag, hence the change from `awk` to `sed`.
* Part of my testing for this stack was to [add a `replace` entry for hcsshim](https://github.com/containerd/containerd/pull/4419/commits/211176521cdba964c96bd7319f1eb28c216f385f) and hence `grep` finds multiple instances of `Microsoft/hcsshim`, hence the addition of `-m 1` to only take the first match.

See 260cb0b46eb497ee5ed70a4ff535ae064854acb4

Reply:
Following up on this, a new hcsshim release v0.8.15 includes the necessary changes, and for some reason this code was not seeing that tag on CI in the `git fetch origin` call, so I've dropped this change for now.

Since I'm using `replaces` for testing elsewhere (I don't need the shim to be updated AFAIK), I suspect I'll only need the `-m 1` part when doing that, but this PR doesn't need this change.